### PR TITLE
Recent transactions list

### PR DESCRIPTION
**Pull request by krisbitney**
related to https://github.com/GoodDollar/GoodCollective/issues/105

### CODE REVIEW

**Comment by krisbitney** [comment]
@sirpy @L03TJ3 Please review this soon. It needs to be merged before https://github.com/GoodDollar/GoodCollective/pull/113 since it adds new locations where user full names need to be fetched.

File: packages/app/src/components/TransactionList/TransactionList.tsx [hunk 1]
```diff
@@ -18,7 +20,7 @@ function TransactionList({ collective }: TransactionListProps) {
   });
   const { navigate } = useCrossNavigate();
 
-  const transactions: Transaction[] = useRecentTransactions(collective, 6);
+  const transactions: Transaction[] = useRecentTransactions(collective, 6, 1000);
 
   const onClickShowMore = () => navigate('/profile/abc123/activity');
```
Comment (L03TJ3) [inline packages/app/src/components/TransactionList/TransactionList.tsx:25]: Where is this supposed to end up on? just the users own profile?

Comment (krisbitney) [reply packages/app/src/components/TransactionList/TransactionList.tsx:25]: it goes to the "activity page" with mock data. You can change it to whatever you want. It's not clear where it's supposed to go. A page showing all transactions and the "activity page" are not mentioned in the original requirements or the original wireframe, and are out of scope for the MVP.

Comment (krisbitney) [reply packages/app/src/components/TransactionList/TransactionList.tsx:25]: please correct me if i'm wrong

**Review by L03TJ3 — APPROVED** [review]

File: packages/app/src/components/TransactionList/TransactionList.tsx (hunk 1 above)
Comment (L03TJ3) [reply packages/app/src/components/TransactionList/TransactionList.tsx:25]: Ok I will verify.
there is bug though where a empty div block is blocking the 'show more' button

image

**Pull request merged**